frontend/src/pages/SignUp.jsx

```diff
@@ -75,3 +80,17 @@ export default function SignUp() {
 		</div>
 	);
 }
+
+function checkValidEmailDomain(email) {
```

Comment:
Replace this with the regex from earlier comment

Reply:
Added regex